cSpell.json

```diff
@@ -0,0 +1,159 @@
+{
+	"version": "0.2",
+	"language": "en",
+	"words": [
+		"Avalonia",
+		"binlog",
+		"Blazor",
+		"blockquotes",
+		"Bootstrapper",
+		"brotli",
+		"browsersync",
+		"Callout",
+		"chatops",
+		"codebases",
+		"Codespace",
+		"Codespaces",
+		"Contoso",
+		"Cupertino",
+		"customizability",
+		"databinding",
+		"datacontext",
+		"datagrid",
+		"devs",
+		"Dismissable",
+		"Docfx",
+		"ellipsize",
+		"Expando",
+		"Framebuffer",
+		"Geolocation",
+		"globbing",
+		"Gyrometer",
+		"Hanja",
+		"hectopascals",
+		"laggy",
+		"layouter",
+		"layouting",
+		"Listview",
+		"LLRM",
+		"mergeable",
+		"MSAL",
+		"msbuild",
+		"MVVM",
+		"NETSDK",
+		"netstandard",
+		"oidc",
+		"Omnisharp",
+		"overscroll",
+		"Packt",
+		"parameterless",
+		"pickable",
+		"Pluralsight",
+		"POSIX",
+		"reentrantly",
+		"roadmap",
+		"sandboxed",
+		"Segoe",
+		"Silverlight",
+		"Skia",
+		"skiasharp",
+		"Storyboarded",
+		"struct",
+		"Syncfusion",
+		"templatable",
+		"templating",
+		"timespan",
+		"Tizen",
+		"TLDR",
+		"triaging",
+		"UI's",
+		"unmanaged",
+		"Uno's",
+		"unoplatform",
+		"UWP's",
+		"VSIX",
+		"WEBASSEMBLY",
+		"walkthrough"
+	],
+	"ignoreWords": [
+		"ADAL",
+		"Authenticode",
+		"automagically",
+		"Divio's",
+		"emcc",
+		"emscripten",
+		"Flatpickr",
+		"Giesel",
+		"GPIO",
+		"hdpi",
+		"Headered",
+		"Hoefling",
+		"HRESULT",
+		"IDBFS",
+		"Infragistics",
+		"jonathanpeppers's",
+		"Krueger",
+		"libgtk",
+		"Liu's",
+		"Matteo",
+		"McCaffrey",
+		"mdpi",
+		"Mergify",
+		"netstd",
+		"nventive",
+		"slnf",
+		"Stetho",
+		"Talkin",
+		"UADO",
+		"Udemy",
+		"winui"
+	],
+	"patterns": [
+		{
+			"name": "Markdown links",
+			"pattern": "\\((.*)\\)",
+			"description": ""
+		},
+		{
+			"name": "Markdown code blocks",
+			"pattern": "/^(\\s*`{3,}).*[\\s\\S]*?^\\1/gmx",
+			"description": "Taken from the cSpell example at https://cspell.org/configuration/patterns/#verbose-regular-expressions"
+		},
+		{
+			"name": "Inline code blocks",
+			"pattern": "\\`([^\\`].*?)\\`",
+			"description": "https://stackoverflow.com/questions/41274241/how-to-capture-inline-markdown-code-but-not-a-markdown-code-fence-with-regex"
+		},
+		{
+			"name": "Link contents",
+			"pattern": "\\<a(.*)\\>",
+			"description": ""
+		},
+		{
+			"name": "Snippet references",
+			"pattern": "-- snippet:(.*)",
+			"description": ""
+		},
+		{
+			"name": "Snippet references 2",
+			"pattern": "\\<\\[sample:(.*)",
+			"description": "another kind of snippet reference"
+		},
+		{
+			"name": "Multi-line code blocks",
+			"pattern": "/^\\s*```[\\s\\S]*?^\\s*```/gm"
+		}
+	],
+	"ignoreRegExpList": [
+		"Markdown links",
+		"Markdown code blocks",
+		"Inline code blocks",
+		"Link contents",
+		"Snippet references",
+		"Snippet references 2",
+		"Multi-line code blocks"
+	],
+	"ignorePaths": [
+		"doc/index.md"
+	]
```

Comment:
Why this one is ignored?

Reply:
Sorry, I had commented the reason for this but it may have disappeared when I force merged.

I ignored this because of the amount of HTML in file caused many false positives and it seemed better to ignore it as a non-markdown file.